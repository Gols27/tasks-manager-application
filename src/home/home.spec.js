import React from 'react';
import { Home } from './';
import { mount } from 'enzyme';
import { mountToJson } from 'enzyme-to-json';


it('renders without crashing', () => {
  const wrapper = mount(<Home getTasks={jest.fn()} addTask={jest.fn()}/>);
  wrapper.find('button').simulate('click');
  expect(mountToJson(wrapper)).toMatchSnapshot();
});