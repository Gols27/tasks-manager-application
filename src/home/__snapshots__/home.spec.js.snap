// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<Home
  addTask={
    [MockFunction] {
      "calls": Array [
        Array [
          Object {
            "status": "Pending",
          },
        ],
      ],
      "results": Array [
        Object {
          "type": "return",
          "value": undefined,
        },
      ],
    }
  }
  getTasks={
    [MockFunction] {
      "calls": Array [
        Array [],
      ],
      "results": Array [
        Object {
          "type": "return",
          "value": undefined,
        },
      ],
    }
  }
>
  <div
    className="tasklist"
  >
    <hr />
    <div
      className="button-container"
    >
      <button
        className="button"
        onClick={[Function]}
        size="lg"
      >
        Add Item
      </button>
    </div>
  </div>
</Home>
`;
